assets/static/js/dataset-submission.js

```diff
@@ -256,6 +279,7 @@ $(function() {
             url: url + "/" + datasetSubmissionId + "?validate=false",
             method: "PUT",
             data: formData,
+            timeout: 30000,
```

Comment:
For these PUT operations, I think 30 seconds is really generous. I'd think what doesn't finish in 5 seconds will likely not finish in 30.  Maybe we should shorten?

Reply:
It's less than the 60 seconds before, but less than 30 would be better, not sure what the magic number would be.